leapp/libraries/stdlib/__init__.py

```diff
@@ -32,3 +90,60 @@ def call(args, split=True):
     if split:
         return r.splitlines()
     return r
+
+
+def _logging_handler(fd_info, buffer):
+    """
+    Log while in a leapp debug mode
```

Comment:
Mention where is the buffer being logged - it is logged to stdout or stderr.

Reply:
As it logs either to STDOUT or STDERR, its description was updated and extended accordingly.